### ISSUE TITLE
FuturePack Crash

### ISSUE THREAD

**Issue by ccfox** [thedarkcolour/Future-MC#53]
Posting here as per request, error log from the crash resulting from using both futurepack and future MC Please let me know if this wasn't what you needed;

```

Description: Initializing game

java.lang.ExceptionInInitializerError
	at futurepack.common.dim.biome.FPBioms.register(FPBioms.java:30)
	at futurepack.common.FPRegistry.registerBiomes(FPRegistry.java:160)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_29_FPRegistry_registerBiomes_Register.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
	at net.minecraftforge.fml.common.eventhandler.EventBus$1.invoke(EventBus.java:144)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182)
	at net.minecraftforge.registries.GameData.fireRegistryEvents(GameData.java:857)
	at net.minecraftforge.fml.common.Loader.preinitializeMods(Loader.java:630)
	at net.minecraftforge.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:252)
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:467)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:378)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
Caused by: java.lang.IllegalArgumentException: Cannot add types to unregistered biome net.minecraft.world.biome.BiomePlains@135064ea
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:191)
	at net.minecraftforge.common.BiomeDictionary.addTypes(BiomeDictionary.java:185)
	at net.minecraftforge.common.BiomeDictionary.registerVanillaBiomes(BiomeDictionary.java:410)
	at net.minecraftforge.common.BiomeDictionary.<clinit>(BiomeDictionary.java:176)
	... 18 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
	at futurepack.common.dim.biome.FPBioms.register(FPBioms.java:30)
	at futurepack.common.FPRegistry.registerBiomes(FPRegistry.java:160)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_29_FPRegistry_registerBiomes_Register.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
	at net.minecraftforge.fml.common.eventhandler.EventBus$1.invoke(EventBus.java:144)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182)
	at net.minecraftforge.registries.GameData.fireRegistryEvents(GameData.java:857)
	at net.minecraftforge.fml.common.Loader.preinitializeMods(Loader.java:630)
	at net.minecraftforge.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:252)
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:467)

-- Initialization --
Details:
Stacktrace:
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:378)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

```

**Comment by thedarkcolour** [comment]
Disable bees in my mod. I replaced a few biomes in order to get beehives to spawn. I am going to find an alternative to this in next update.

**Comment by thedarkcolour** [comment]
Fixed in 0.1.12
